packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -251,6 +249,9 @@ abstract contract FundingRateApplier is FeePayer {
                 }
             }
         }
+    }
+
+    function _getLatestFundingRate() internal view returns (FixedPoint.Signed memory) {
```

Comment:
Can this function be removed, and just replace any `_getLatestFundingRate` calls with `fundingRate.rate`?

Reply:
sure. I've removed this function as it's only used once.